### DISCUSSION TITLE
Flink CDC community version docs

### DISCUSSION THREAD

**Discussion by tatuke** [apache/flink-cdc#3234] [General] [answered]
It seems all flink-cdc docs and guide were integrated on 
https://nightlies.apache.org/flink/flink-cdc-docs-release-3.0/ and [https://ververica.github.io](https://ververica.github.io/flink-cdc-connectors/release-2.1/content/quickstart/index.html) were closed.But version docs just 3.0-rs could be read.
![image](https://github.com/apache/flink-cdc/assets/46305821/513f2612-6247-4f0a-b9ad-f7ff2ad39539)
What if i want to know about previous version 2.2 for example.and i can't find any guide.Is there any path to get previous version docs? I want to contiue my work on these old version. I also have some questions about data-format options (why changelog-json doesn't supported after 2.2?) but i will open a new discussions or jira.Again Thanks for your amazing work!

**Comment by yuxiqian** [comment, marked as answer]
Hi @tatuke, legacy documents are not transferred to apache.org when Flink CDC was donated to ASF. However, all documents are still available in GitHub release branches. For example, if you're looking for docs for release-2.2, you may switch to specific branch with [this link](https://github.com/apache/flink-cdc/tree/release-2.2/docs/content) and navigate to specific [Markdown file like this](https://github.com/apache/flink-cdc/blob/release-2.2/docs/content/formats/changelog-json.md). They should provide you the same information as the legacy site does.

**Reply by tatuke** [reply]
Thank you.
